arcgis-ios-sdk-samples/Maps/Create and save a map/CreateSaveMapViewController.swift

```diff
@@ -34,8 +34,10 @@ private extension UIImage {
 
 class CreateSaveMapViewController: UIViewController, CreateOptionsViewControllerDelegate, SaveAsViewControllerDelegate {
     @IBOutlet private weak var mapView: AGSMapView!
+    @IBOutlet weak var saveButton: UIBarButtonItem!
```

Comment:
```suggestion
    @IBOutlet private weak var saveButton: UIBarButtonItem!
```

I saw you unified all storyboard views to private weak in the other changes, so let's keep them consistent. Ideally they would be `@IBOutlet var`